packages/medusa/src/commands/develop.js

```diff
@@ -1,16 +1,43 @@
+import boxen from "boxen"
 import path from "path"
 import { execSync } from "child_process"
 import spawn from "cross-spawn"
 import chokidar from "chokidar"
+import Store from "medusa-telemetry/dist/store"
 
 import Logger from "../loaders/logger"
 
+const defaultConfig = {
+  padding: 5,
+  borderColor: `blue`,
+  borderStyle: `double`,
+}
+
 export default async function ({ port, directory }) {
   const args = process.argv
   args.shift()
   args.shift()
   args.shift()
 
+  process.on("SIGINT", () => {
+    const configStore = new Store()
+    const hasPrompted = configStore.getConfig("star.prompted") ?? false
+    if (!hasPrompted) {
+      const defaultMessage =
+        `✨ Thanks for using Medusa. ✨\n` +
+        `If you liked it, please consider starring us on GitHub\n` +
```

Comment:
suggestion(non blocking): Maybe adding a new line here between the title and the content